src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/workingLists.types.js

```diff
@@ -15,24 +15,21 @@ import type {
     Sort,
 } from '../../../ListView';
 
-export type WorkingListTemplate = {|
+export type WorkingListTemplate = {
     id: string,
     isDefault?: ?boolean,
     name: string,
-    displayName: string,
-    filters: Object,
     access: {
-        read: boolean,
         update: boolean,
         delete: boolean,
         write: boolean,
         manage: boolean,
     },
-    notPreserved?: ?boolean,
-    nextEventQueryCriteria?: Object,
+    notPreserved?: boolean,
+    updating?: boolean,
     deleted?: boolean,
     skipInitDuringAddProcedure?: boolean,
-|};
+};
```

Comment:
What makes you want to remove the `||` from the annotation?

And also perhaps you can explain to me how this `template` is different conceptually from the `types/eventWorkingLists.types.js` one

Reply:
The `criteria` and `nextCriteria` props are not used in the `WorkingLists` and are therefore not required here. They are required in the `EventWorkingLists` though. We could remove them when passing the data to the `WorkingLists`, but opted to be pragmatic here and pass it through to the `WorkingLists` (if not we would need to readd them when callbacks are triggered)

Reply:
Okay thanks for explaining. Do we want to annotate strictly here?

Reply:
The object contains more props (`criteria` and `nextCriteria`), so can't do exact annotation here.